src/components/Layout/index.tsx

```diff
@@ -67,7 +66,7 @@ const Layout: React.FC<Props> = ({ children, title, description }) => {
         logosmall={logoSmall}
         host={HOST_URL}
         community={COMMUNITY_URL}
-        donateurl="/donate"
+        donateURL="/donate"
```

Comment:
I changed it to lowercase on the `packages` project due to it was not working with camelcase, I noticed this while adding the `logosmall` prop, when I tried to set it as `logoSmall` it was not working, this may be related to HTML attributes are case insensitive

Reply:
This is odd as for me wasn't working with lowercase :/

Reply:
you may have to execute yarn upgrade, I updated the version for the header-component a few days ago